CPPLINT.cfg

```diff
@@ -0,0 +1,3 @@
+# tabs should be tab, not space in bsf.
+# braces should be preceded by a newline
+filter=-whitespace/tab,-whitespace/braces
```

Comment:
Is clang-format a perhaps a better option? Haven't really used cpplint, but some IDE's I work with already integrate clang-format.